source/OVERVIEW.rst

```diff
@@ -21,13 +21,20 @@ Jupyter Notebooks and Markdown files into publishable content.
 Providing online-executable runtimes and a powerful data caching system, NeuroLibre makes a **groundbreaking preprint
 server where you can plant the seeds of living publications.**
 
+.. seealso:: If you would like to submit a NeuroLibre preprint, please read our `submission guidelines <https://docs.neurolibre.com/en/latest/SUBMIT.html>`_.
+
+As an author and before submitting, you:
+
+1. Create a github repository with notebooks that you would like to share.
+2. Make your work reproducible by using `binder configuration files <https://mybinder.readthedocs.io/en/latest/using/config_files.html>`_
+3. (optionnaly) Test your repository on `binder <https://mybinder.org/>`_.
+4. Design a `Jupyter Book <https://jupyterbook.org/intro.html>`_ with all relevant bibliography.
```

Comment:
Add Jupyter Book configurations and your bibliography:

Reply:
I did not want to make this list huge, and for me bibliogrpahy is part of the writing (i.e. jupyter book "design"). Can you think of something that takes no more than one line ?